cli/src/cmd/forge/snapshot.rs

```diff
@@ -218,7 +218,9 @@ impl FromStr for SnapshotEntry {
                             Some(SnapshotEntry {
                                 contract_name: file.as_str().to_string(),
                                 signature: sig.as_str().to_string(),
-                                gas_used: TestKindGas::Standard(gas.as_str().parse().unwrap()),
+                                gas_used: TestKindReport::Standard {
```

Comment:
nit: If we rename this to `TestKindReport`, we should also move away from `gas_used` to something that is more in line with `TestKindReport`